boa_cli/src/main.rs

```diff
@@ -136,6 +160,22 @@ enum DumpFormat {
     JsonPretty,
 }
 
+#[derive(Debug, Clone, Copy, ValueEnum)]
+enum FlowgraphFormat {
+    /// Generates in graphviz format.
+    Graphviz,
+    /// Generates in mermaid format.
+    Mermaid,
+}
+
+#[derive(Debug, Clone, Copy, ValueEnum)]
+enum FlowgraphDirection {
+    TopToBottom,
+    BottomToTop,
+    LeftToRight,
+    RightToLeft,
+}
```

Comment:
I would add some enum-level and variant level documentation here, even if it's pretty much clear what it does.